lib/miq_automation_engine/models/miq_ae_password.rb

```diff
@@ -31,6 +31,10 @@ def self.v1_key
     MiqPassword.v1_key
   end
 
+  def self.all_keys
```

Comment:
Since inside MiqPassword, it references 'class.x', we need to make sure that `MiqAePassword` responds to the same methods.
Alternative, we could hard code 'MiqPassword.` in MiqPassword instead of `self.class.`